src/Stratis.Bitcoin.Features.SmartContracts/Networks/SmartContractsRegTest.cs

```diff
@@ -39,7 +39,8 @@ public SmartContractsRegTest()
                 maxBlockBaseSize: 1_000_000,
                 maxStandardVersion: 2,
                 maxStandardTxWeight: 100_000,
-                maxBlockSigopsCost: 20_000
+                maxBlockSigopsCost: 20_000,
+                provenHeadersActivationHeight: 10_000_000 // TODO: Set it to the real value once it is known.
```

Comment:
SC wont have proven headers so this could just be zero but no big deal this will be fixed later.